src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -57,6 +57,7 @@ public net.minecraft.client.multiplayer.ClientAdvancements f_104390_ # advanceme
 public net.minecraft.world.level.block.DoorBlock m_245755_(Lnet/minecraft/world/entity/Entity;Lnet/minecraft/world/level/Level;Lnet/minecraft/core/BlockPos;Z)V # playSound
 
 public net.minecraft.world.entity.Entity f_19816_ # eyeHeight
+public net.minecraft.world.entity.Entity m_284535_(Lnet/minecraft/world/level/Level;)V # setLevel
```

Comment:
Plain no, unlike BE entity level is notnull, please find another way

Reply:
There's no harm in it, I tested it.  The entity is never added to any world, it only exists for the renderer and it does get a level set while that happens.

Reply:
Not even BE has this method nullable (few vanilla BEs), so at least try catch it. Plus I really don't like it, unlike BE entities simply have non-null level contract..